.github/workflows/release.yml

```diff
@@ -160,6 +160,8 @@ jobs:
         with:
           node-version: 18.x
           cache: 'npm'
+      - name: pip
+        run: pip install setuptools
```

Comment:
What does setup tools do?

Reply:
So there were a handful of problems that this PR was solving. One of them being that we couldn't build a Windows or a Mac binary anymore. The Mac problem was due to dependency issues. Basically npm install would fail on a Mac before it even got to building a binary.

After the dependency issues were resolved, Mac binary built fine but Windows still had failures. One of the errors was that the distutils (dist folder) python modules had been "end of life". Basically, an alternative to using distutils was to install setuptools. After installing setuptools the Windows binary built.

https://www.geeksforgeeks.org/differences-between-distribute-distutils-setuptools-in-python/

Reply:
Does this information need to be updated to any of our documentation for ZAP's source installation ?

Reply:
yes, probably